### ISSUE TITLE
Can't rotate M8C screen in linux

### ISSUE THREAD

**Issue by Szabrownick** [laamaa/m8c#171]
I tried everything to rotate M8C in linux using Lite OS. Nothing works. Is it any option in settings to rotate M8C?
cmdline.txt, config.txt, fkms, and any other options not working. tested on Waveshare 3,5 LCD HDMI and KEDEI 3,5 LCD HDMI

**Comment by laamaa** [comment]
No, handling display orientation should generally happen on some other level than application... you could try to check if the displays have drivers available somewhere, or try to change orientation with xrandr or something like that :)
